packages/manager/apps/hpc-vmware-managed-vcd/src/utils/getPricedOrderableResource.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  IVdcOrderableVHost,
+  IVdcOrderableVhostPriced,
+} from '@/types/vcd-vdc-orderable-resource.interface';
+import { TVcdCatalog } from '@/types/vcd-catalog.interface';
+
+export const getVhostUcentsPrice = (product: IVdcOrderableVhostPriced) =>
+  product.prices[0]?.priceInUcents;
+
+export const getPricedOrderableVhostList = ({
+  vhostList,
+  catalog,
+}: {
+  vhostList: IVdcOrderableVHost[];
+  catalog: TVcdCatalog;
+}): IVdcOrderableVhostPriced[] => {
+  if (!vhostList || !catalog) {
+    return [];
+  }
+  return vhostList
+    .map((vhost) => {
+      const product = catalog.find((pdct) => pdct.planCode === vhost.profile);
+      return { ...vhost, prices: product?.prices || [] };
+    })
```

Comment:
Should not be a filter instead of a map here ? Only available vHosts should be displayed

Reply:
I don't know if an orderable vHost could be missing in the catalog, but I'll filter to be sure! 👍